api/room/models.py

```diff
@@ -24,12 +33,14 @@ class Room(Base, Utility):
     wing_id = Column(Integer, ForeignKey('wings.id'))
     cancellation_duration = Column(Integer, default=10)
     floor = relationship('Floor')
-    resources = relationship(
-        'Resource', cascade="all, delete-orphan",
-        order_by="func.lower(Resource.name)")
+    resources = relationship('Resource', cascade="all, delete-orphan")
     events = relationship('Events', cascade="all, delete-orphan")
     response = relationship('Response', cascade="all, delete-orphan")
-    devices = relationship('Devices', cascade="all, delete-orphan")
+    room_tags = relationship(
```

Comment:
Hey @bevkololi , could there be a reason as to why the devices field was removed? If so, kindly throw some light.

Reply:
@nakatuddesuzan, it has been updated now.